app/utils/navigation/index.ts

```diff
@@ -1,14 +1,16 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {IntlShape} from 'react-intl';
 import {Alert} from 'react-native';
 import {Navigation, Options} from 'react-native-navigation';
 
 import {Screens, ServerErrors} from '@constants';
 import {isServerError} from '@utils/errors';
 
-export const appearanceControlledScreens = new Set([
+import type {AvailableScreens} from '@typings/screens/navigation';
+import type {IntlShape} from 'react-intl';
+
+export const appearanceControlledScreens = new Set<AvailableScreens>([
     Screens.ONBOARDING,
```

Comment:
Probably not for this PR (already big enough) but now that we have the `AvailableScreens` type, I wonder if we should stop using the `Screens` constant and just rely on type checks. 0/5.